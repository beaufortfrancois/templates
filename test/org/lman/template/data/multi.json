{
  "baz": "blah",
  "flats1": [
    { "foo": 1, "bar": 2 },
    { "foo": 3, "bar": 4 }
  ],
  "flats2": [
    { "foo": 5, "bar": 6 },
    { "foo": 7, "bar": 8 }
  ]
}
