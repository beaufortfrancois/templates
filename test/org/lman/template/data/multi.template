{{baz}} -
{{#flats1}} foo: {{@.foo}}, bar: {{@.bar}};{{/flats1}} -
{{#flats2}} foo: {{@.foo}}, bar: {{@.bar}};{{/flats2}}
