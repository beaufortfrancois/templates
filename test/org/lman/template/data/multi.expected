blah -
 foo: 1, bar: 2; foo: 3, bar: 4; -
 foo: 5, bar: 6; foo: 7, bar: 8;
